### PR TITLE
Cloud Native Security White Paper Italian translation

### PR DESCRIPTION
**Pull request by paolomainardi**
Refs https://github.com/cncf/tag-security/issues/1014: Add Italian translation

Signed-off-by: Paolo Mainardi <paolo.mainardi@sparkfabrik.com>

Many, many, many thanks to @annalisag-spark for making this happen

### CODE REVIEW

File: security-whitepaper/v2/cloud-native-security-whitepaper-it.md [hunk 1]
```diff
@@ -0,0 +1,1335 @@
...
+
+raja.faisal@gmail.com, Savitha Raghunathan (Red Hat), Sayantani Saha (Independent)
+
+- **Reviewers**: Jon Zeolla, @nyrahul (Accuknox), Ragashree Shekar, Steven Hadfield, Kapil Bareja, Malini Bhandaru,
+
+Mikko Ylinen, Jonah Kowall (@jkowall), Kuang Dahu,Ariel Shuper, Eric Li (AlibabaCloud), Tanner Randolph (Applied
+
+Systems), Mark Dalton Gray (Microsoft), Ciara Carey, Brandon Lum (TAG Security Chair), Emily Fox (TOC Liaison) and
+
+Justin Cormack (TOC liaison).
```
Comment (mrcdb) [inline security-whitepaper/v2/cloud-native-security-whitepaper-it.md:25]: the markdown rendering of these lists is different from the original whitepaper due to the newlines.

**Comment by lumjjb** [comment]
Woo! Exciting!

File: security-whitepaper/v2/cloud-native-security-whitepaper-it.md (hunk 1 above)
Comment (paolomainardi) [reply security-whitepaper/v2/cloud-native-security-whitepaper-it.md]: fixed thanks

File: security-whitepaper/v2/cloud-native-security-whitepaper-it.md [hunk 2]
```diff
@@ -0,0 +1,1335 @@
...
+
+**Version 2 in Italian (December 2022)**
+
+**Translators and reviewers**: Paolo Mainardi (SparkFabrik, TAG Security), Daniele Monti (SparkFabrik), Andrea Panisson (SparkFabrik), Giuseppe Arancio (SparkFabrik), Filippo Merante Caparrotta (SparkFabrik), Marcello Testi (SparkFabrik), Cartografos), Annalisa Gennaro (SparkFabrik, Glossary, Cartografos, KCD)
+
+# Executive Summary
```
[the remark this answers is not in the document]
Comment (paolomainardi) [reply security-whitepaper/v2/cloud-native-security-whitepaper-it.md]: thanks,@mrcdb, for your review!
I've just noticed that the headings are very much different; I'm going to fix them to let them match as close as possible with English version one; it seems that something has been changed when we converted google doc to markdown

**Comment by paolomainardi** [comment]
I should have fixed all headings and leftovers of automatic conversion to markdown; let me know if you find other occurrences that should be fixed.

**Comment by paolomainardi** [comment]
I see the CI jobs are not very happy, especially limiting, but I am quite sure that the format is almost identical to the original version, have to double check it tho.

**Comment by paolomainardi** [comment]
Ok got it, I added the lines of the disabled rules that were missing, not it seems to be on par with the original one, with the same errors.

**Comment by PushkarJ** [comment]
Thanks a ton @paolomainardi for the PR. I will continue to keep an eye on CI jobs but in the meantime can you squash the commits & sign the commit so DCO check is happy?

**Comment by PushkarJ** [comment]
Looked at the CI checks. You can ignore spelling and links errors. 

What do you think of lint issues? Are those fixable in your opinion?

**Comment by paolomainardi** [comment]
@PushkarJ, thanks for your review! Just fixed the linting issues (also in the English version) and i should have signed-off the rebased commit too.

File: security-whitepaper/v2/cloud-native-security-whitepaper-it.md [hunk 3]
```diff
@@ -0,0 +1,1153 @@
...
+
+[^1]: Un ulteriori modello da prendere in considerazione è Cloud, Clusters, Containers e Code: [https://kubernetes.io/docs/concepts/security/overview/](https://kubernetes.io/docs/concepts/security/overview/)
+
+[^2]: Esempio - [MITRE ATT&CK Framework for Kubernetes](https://www.darkreading.com/threat-intelligence/microsofts-kubernetes-threat-matrix-heres-whats-missing/a/d-id/1339106)
+
+[^3]: [Shifting security left](https://www.devsecops.org/blog/2016/5/20/-security) spesso porta le organizzazioni a perdere di vista il monitoraggio della sicurezza operativa. È importante che la sicurezza esista in tutte le parti del ciclo di vita e che le organizzazioni valutino continuamente altri aspetti dei loro processi aziendali e tecnologici per abbracciare la sicurezza in quanto elemento culturale e buona abitudine.
+
+[^4]: Il capitale umano è un asset vitale e necessario per il successo di qualsiasi organizzazione, la corrispondente proprietà intellettuale e il capitale relazionale devono essere protetti.
+
+[^5] [https://blog.aquasec.com/malicious-container-image-docker-container-host](https://blog.aquasec.com/malicious-container-image-docker-container-host)
```
Comment (mrcdb) [inline security-whitepaper/v2/cloud-native-security-whitepaper-it.md:1135]:
```suggestion
[^5]: [https://blog.aquasec.com/malicious-container-image-docker-container-host](https://blog.aquasec.com/malicious-container-image-docker-container-host)
```

File: security-whitepaper/v2/cloud-native-security-whitepaper-it.md [hunk 4]
```diff
@@ -0,0 +1,1153 @@
...
+
+[^6]: Secondo Applied Software Measurement, Capers Jones, 1996 e tenendo conto dell'inflazione, l'85% dei difetti viene introdotto durante la scrittura del codice con un costo di correzione di $ 41 rispetto a un costo di correzione post rilascio di $ 26.542.
+
+[^7]: cisecurity.org maintains a listing of benchmarks for hardening
+
+[^8] È fondamentale notare che mentre l'autenticazione è disponibile per l'uso, la [mutual authentication](https://github.com/PushkarJ/tag-security/blob/cnswpv2/security-whitepaper/v1/cloud-native-security-whitepaper.md#zero-trust-architecture) è il meccanismo preferito non solo per verificare il client ma anche il server (outsider contro insider).
```
Comment (mrcdb) [inline security-whitepaper/v2/cloud-native-security-whitepaper-it.md:1141]:
```suggestion
[^8]: È fondamentale notare che mentre l'autenticazione è disponibile per l'uso, la [mutual authentication](https://github.com/PushkarJ/tag-security/blob/cnswpv2/security-whitepaper/v1/cloud-native-security-whitepaper.md#zero-trust-architecture) è il meccanismo preferito non solo per verificare il client ma anche il server (outsider contro insider).
```

**Review by mrcdb — COMMENTED** [review]
@paolomainardi thanks for all the changes! I could only see a couple more rendering issues in the references section, the rest looks great to me.

**Comment by PushkarJ** [comment]
Lint CI job passed - yay 🎉

**Comment by lumjjb** [comment]
We'll keep this up for comments for another week since we just shared it around the network, then should be good to merge

**Comment by paolomainardi** [comment]
Ok thanks @lumjjb

**Comment by paolomainardi** [comment]
Hi @lumjjb, and happy new year to everyone.

I was coming back here to ask for updates about this PR and if it's finally good to merge, let me know if I can do something to help, thanks.

**Comment by paolomainardi** [comment]
Just asking the reviewers to publish their comments on these Markdown files, I spent hours reversing back the Drive document to a readable markdown version, it seemed easy to the beginning.

Thanks!

File: security-whitepaper/v2/cloud-native-security-whitepaper-it.md [hunk 5]
```diff
@@ -0,0 +1,1153 @@
...
+
+#### Distribuzione
+
+La sicurezza della supply chain del software è particolarmente critica nei modelli che consentono iterazioni sempre più rapide. I cicli di vita delle applicazioni Cloud Native devono includere metodi per verificare non solo l'integrità del workload stesso, ma anche il processo per la creazione del workload e della modalità di funzionamento. Questa sfida è amplificata dall’uso necessario, funzionale e coerente di software open source e runtime di terze parti, inclusi i livelli delle dipendenze a monte. Artefatti, come container images, presenti nella pipeline del ciclo di vita richiedono una scansione automatizzata continua e aggiornamenti per garantire la sicurezza da vulnerabilità, malware, pratiche di sviluppo non sicure e altri potenziali rischi. Dopo aver completato questi controlli, è importante firmare crittograficamente gli artefatti per garantirne l'integrità e imporre il non-ripudio. Degni di nota per una distribuzione sicura sono anche l’immagine in codice binario e l'URL dell'immagine immutabili.
```
Comment (mrcdb) [inline security-whitepaper/v2/cloud-native-security-whitepaper-it.md:44]:
```suggestion
La sicurezza della supply chain del software è particolarmente critica nei modelli che consentono iterazioni sempre più rapide. I cicli di vita delle applicazioni Cloud Native devono includere metodi per verificare non solo l'integrità del workload stesso, ma anche il processo per la creazione del workload e della modalità di funzionamento. Questa sfida è amplificata dall’uso necessario, funzionale e coerente di software open source e runtime di terze parti, inclusi i livelli delle dipendenze a monte. Artefatti, come container images, presenti nella pipeline del ciclo di vita richiedono una scansione automatizzata continua e aggiornamenti per garantire la sicurezza da vulnerabilità, malware, pratiche di sviluppo non sicure e altri potenziali rischi. Dopo aver completato questi controlli, è importante firmare crittograficamente gli artefatti per garantirne l'integrità e imporre il non-ripudio. Degni di nota per una distribuzione sicura sono anche la creazione di binari immutabili per le immagini e di URL immutabili per le stesse.
```

File: security-whitepaper/v2/cloud-native-security-whitepaper-it.md [hunk 6]
```diff
@@ -0,0 +1,1153 @@
...
+
+#### Deployment
+
+La sicurezza integrata lungo le fasi di sviluppo e distribuzione permette la validazione continua e in tempo reale degli attributi del workload, come ad esempio che gli artefatti firmati siano verificati, che le politiche di sicurezza della container image e del runtime siano rispettate, che l'idoneità e l'affidabilità dell'hosting siano convalidate tramite policy di binary authorization negli ambienti di staging e di produzione. I controlli nella fase di deployment rappresentano l'ultima possibilità di convalidare, correggere e applicare queste verifiche prima che il workload inizi a erogare il servizio delle specifiche esigenze di business per cui è stato concepito. Funzionalità sicure di osservabilità del workload, implementate insieme al workload stesso, consentono di monitorare i log e le metriche disponibili con un elevato livello di confidenza, completando gli aspetti di sicurezza integrata.
```
Comment (mrcdb) [inline security-whitepaper/v2/cloud-native-security-whitepaper-it.md:48]:
```suggestion
La sicurezza integrata lungo le fasi di sviluppo e distribuzione permette la validazione continua e in tempo reale degli attributi del workload, come ad esempio che gli artefatti firmati siano verificati, che le politiche di sicurezza della container image e del runtime siano rispettate, che l'idoneità e l'affidabilità dell'hosting siano convalidate tramite policy di binary authorization negli ambienti di staging e di produzione. I controlli nella fase di deployment rappresentano l'ultima possibilità di convalidare, correggere e applicare queste verifiche prima che il workload inizi a erogare il servizio delle specifiche esigenze di business per cui è stato concepito. Funzionalità di osservabilità sicura del workload, implementate insieme al workload stesso, consentono di monitorare i log e le metriche disponibili con un elevato livello di confidenza, completando gli aspetti di sicurezza integrata.
```

File: security-whitepaper/v2/cloud-native-security-whitepaper-it.md [hunk 7]
```diff
@@ -0,0 +1,1153 @@
...
...le prendere in considerazione una sandbox basata su macchine virtuali. Per il trattamento di dati finanziari sensibili, potrebbe valere la pena considerare un ambiente di esecuzione affidabile (hardware dedicato alla crittografia per processo o per macchina virtuale) così come i Confidential Containers. All'interno del tipico ambiente di runtime Cloud Native, le applicazioni sono spesso composte da diversi microservizi indipendenti e con un unico scopo, che comunicano tra loro tramite il service layer reso possibile dall’orchestrazione dei container. Le best practice a protezione di questa architettura di componenti correlati garantiscono che solo i processi approvati operino all'interno di un _namespace_ del container, impediscono e alla peggio allertano in caso di tentativi di accesso non autorizzato alle risorse e monitorano il traffico di rete per rilevare l'attività di intrusi ostili. Service Mesh è un'altra astrazione che fornisce funzionalità consolidate e complementari per servizi orchestrati, senza imporre modifiche al software del workload (ad es. logging del traffico API, crittografia del livello di trasporto, tagging dell’osservabilità, autenticazione e autorizzazione).
```
Comment (mrcdb) [inline security-whitepaper/v2/cloud-native-security-whitepaper-it.md:52]:
```suggestion
L'ambiente di runtime Cloud Native può essere esso stesso suddiviso in livelli di componenti correlati con distinti problemi di sicurezza[^1], ad es. hardware, provider, sistema operativo, rete, storage, container image runtime, orchestrazione. Il container runtime è costituito da diverse implementazioni per vari livelli di isolamento, ad es. kernel condiviso, micro-vm sandbox e trusted execution environment sandbox (sandbox dell'ambiente di esecuzione affidabile). È fondamentale scegliere un runtime che soddisfi i requisiti di sicurezza previsti. Ad esempio, per un workload inaffidabile in esecuzione in un ambiente multi-tenant, è possibile prendere in considerazione una sandbox basata su macchine virtuali. Per il trattamento di dati finanziari sensibili, potrebbe valere la pena considerare un ambiente di esecuzione affidabile (hardware dedicato alla crittografia per processo o per macchina virtuale) così come i Confidential Containers. All'interno del tipico ambiente di runtime Cloud Native, le applicazioni sono spesso composte da diversi microservizi indipendenti e dedicati a specifici scopi, che comunicano tra loro tramite il service layer reso possibile dall’orchestrazione dei container. Le best practice a protezione di questa architettura di componenti correlati garantiscono che solo i processi approvati operino all'interno di un _namespace_ del container, impediscono e alla peggio allertano in caso di tentativi di accesso non autorizzato alle risorse e monitorano il traffico di rete per rilevare l'attività di intrusi ostili. Service Mesh è un'altra astrazione che fornisce funzionalità consolidate e complementari per servizi orchestrati, senza imporre modifiche al software del workload (ad es. logging del traffico API, crittografia del livello di trasporto, tagging dell’osservabilità, autenticazione e autorizzazione).
```

File: security-whitepaper/v2/cloud-native-security-whitepaper-it.md [hunk 8]
```diff
@@ -0,0 +1,1153 @@
...
...i di sicurezza tradizionali, integrando al contempo i concetti moderni di volatilità, distribuzione e immutabilità. In questi ambienti in rapida evoluzione, soggetti a rapide iterazioni, è necessaria l'automazione in linea con la pipeline di sviluppo per ottenere risultati sicuri. Le organizzazioni dovrebbero adottare rapidamente questi concetti Cloud Native per supportare la sicurezza orientata al valore durante il loro Cloud Native Journey. Integrando la sicurezza il prima possibile durante tutto il ciclo di vita dello sviluppo, o anche prima con la formazione interattiva degli sviluppatori, le organizzazioni potrebbero attivare un approccio di sicurezza preventiva piuttosto che di sicurezza reattiva (vedi anche "[9 Box of Controls](https://blogs.blackberry.com/en/2016/03/the-9-box-of-controls)"). Si consiglia vivamente alle organizzazioni di valutare lo stack di sicurezza rispetto ai framework di attacco pertinenti[^2]per chiarire quali siano le minacce coperte. Inoltre, le organizzazioni, devono adottare approcci e metodologie che permettano un _shift left[^3]_ della sicurezza, abilitino DevOps e siano sufficientemente flessibili per lavorare con i futuri progressi tecnologici.
```
Comment (mrcdb) [inline security-whitepaper/v2/cloud-native-security-whitepaper-it.md:56]:
```suggestion
La sicurezza Cloud Native cerca di garantire le stesse, se non superiori, condizioni di diligenza, integrità, affidabilità e prevenzione delle minacce dei modelli di sicurezza tradizionali, integrando al contempo i concetti moderni di volatilità, distribuzione e immutabilità. In questi ambienti in rapida evoluzione, soggetti a rapide iterazioni, è necessaria l'automazione in linea con la pipeline di sviluppo per ottenere risultati sicuri. Le organizzazioni dovrebbero adottare rapidamente questi concetti Cloud Native per supportare la sicurezza orientata al valore durante la loro transizione al Cloud Native. Integrando la sicurezza il prima possibile durante tutto il ciclo di vita dello sviluppo, o anche prima con la formazione interattiva degli sviluppatori, le organizzazioni potrebbero attivare un approccio di sicurezza preventiva piuttosto che di sicurezza reattiva (vedi anche "[9 Box of Controls](https://blogs.blackberry.com/en/2016/03/the-9-box-of-controls)"). Si consiglia vivamente alle organizzazioni di valutare lo stack di sicurezza rispetto ai framework di attacco pertinenti[^2]per chiarire quali siano le minacce coperte. Inoltre, le organizzazioni, devono adottare approcci e metodologie che permettano un _shift left[^3]_ della sicurezza, abilitino DevOps e siano sufficientemente flessibili per lavorare con i futuri progressi tecnologici.
```

**Review by mrcdb — COMMENTED** [review]
Suggesting some (minor) changes to the "Fasi del ciclo di vita" section.

**Comment by paolomainardi** [comment]
To everyone asking access to the document, the right one is this: https://docs.google.com/document/u/0/d/1xKZtUkYHD3KDWrE2cIr2I8fVnjMePdhuH7uR3ZbfH6c/mobilebasic

**Comment by paolomainardi** [comment]
Just a note to everyone: https://github.com/cncf/tag-security/pull/1015#issuecomment-1383199464

Please, leave comments here as a patch to markdown; I've already approved some other comments, and the doc version is misaligned.

**Comment by paolomainardi** [comment]
Just integrated the comments from @Vombato and his team, I think we are ready to go now @lumjjb

**Comment by mrcdb** [comment]
@paolomainardi can you add my name to the list of reviewers/translators of the doc, please? :smile:  my affiliation is ControlPlane

**Comment by paolomainardi** [comment]
ciao @mrcdb you can add it with a PR against this branch: https://github.com/paolomainardi/tag-security/tree/patch-2

**Comment by mrcdb** [comment]
@paolomainardi PR done (https://github.com/paolomainardi/tag-security/pull/2)

**Comment by paolomainardi** [comment]
@mrcdb thanks a lot, merged.

**Comment by paolomainardi** [comment]
I guess reviews are terminated, and @lumjjb, we are good to go here.

**Comment by paolomainardi** [comment]
@PushkarJ tried to fix the DCO, everything is signed, but the merge commits are. I hope it is enough to have it merged :)

**Comment by paolomainardi** [comment]
Ok DCO is green now!

**Review by PushkarJ — APPROVED** [review]
Thank you everyone for making this happen

**Review by lumjjb — APPROVED** [review]
LGTM! Thanks so much for your work!

**Pull request merged**